devimint/src/bin/faucet.rs

```diff
@@ -22,7 +22,7 @@ struct Cmd {
     #[clap(long, env = "FM_CLN_SOCKET")]
     cln_socket: String,
     #[clap(long, env = "FM_CONNECT_STRING")]
-    connect_string: Option<String>,
+    invite_string: Option<String>,
```

Comment:
Can we always "invite code"? Having both "invite code" and "invite string" just invites confusion. Also, can you update the `FM_CONNECT_STRING` to `FM_INVITE_CODE`? Also, we have a `client-connect` file that save the invite code in devimint. Can we call that `invite-code`? Let's make sure we change it everywhere.

Reply:
Fedimint only ever calls this `connect_string`, I don't see any results for `connect_code` or `invite_code`
<img width="852" alt="Screenshot 2023-08-01 at 4 28 53 PM" src="https://github.com/fedimint/fedimint/assets/649992/8e0c7287-008d-4dcf-b557-964df21b7a77">